x-pack/plugins/security_solution/public/common/hooks/eql/helpers.ts

```diff
@@ -120,35 +181,3 @@ export const getInterval = (range: Unit, relativeNow: number): EqlAggBuckets =>
       throw new Error('Invalid time range selected');
```

Comment:
You want a range error here?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RangeError

Reply:
Good call, updated!